### ISSUE TITLE
Jump to dictionary entry by double tap

### ISSUE THREAD

**Issue by dolour** [AndBible/and-bible#361]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when i have to toggle between strongs numbers on/off to access the dictionary.

**Describe the solution you'd like**
Make the corresponding words hyperlinks when numbers are toggled off.

**Describe alternatives you've considered**
Its just a very handy feature i greatly enjoyed in other apps, couldn't think of a 3rd alternative.

**Additional context**
Just please, dont highlight those word-hyperlinks for readability reasons. ;)

**Comment by tuomas2** [comment]
Perhaps tapping instead of double-tapping and showing some visual hint that word is clickable should be there. This word-clicking for strongs-enabled bible documents could/should be optional feature and might replace current way of showing strongs numbers (?).

**Comment by machshev** [comment]
I used to use AndBible since it started. But then moved away to AndBibleLexicon mainly because this feature didn't exist, and they had done it so much better.

When the strong numbers are interleaved in the text, it makes it really difficult to read. When they are not, there is no way of looking up a word in the Lexicon. It's worse in Original language texts where the main text is RTL and the strongs are LTR, it is quite mentally jaring to have both shown at the same time.

I'm starting using AndBible again, it's grown a lot since I used it last. AndBibleLexicon is really good for bible study from a UI point of view, but doesn't seem to be open source :/

I'm wondering if this feature is something I could help add.
